### DISCUSSION TITLE
skip build fail

### DISCUSSION THREAD

**Discussion by algoscienceacademy** [skiptools/skip#147] [General]
![skipkey](https://github.com/skiptools/skip/assets/76747815/cb6b0973-6173-45c4-b71a-2d7f656d1f5b)

through i have a 30 days free licence . its don't buid . the xcode project says this here
![final](https://github.com/skiptools/skip/assets/76747815/c2f85d1e-0004-4a5f-af9e-46d4cc5d4c6c)

**Comment by marcprux** [comment]
You need to remove the "#" from the beginning of the "SKIPKEY" line. That denotes a comment in the YAML format.
